azure-pipelines.yml

```diff
@@ -278,7 +278,7 @@ stages:
         -TsaRepositoryName "Arcade-Services"
         -TsaCodebaseName "Arcade-Services"
         -TsaPublish $True'
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), in(variables['Build.SourceBranchName'], 'master', 'production'))}}:
```

Comment:
This is technically incorrect, because this will also trigger if I push a branch called test/master or pizza/production. This is probably fine though.

Reply:
That's fair. Might as well guard against that: https://github.com/dotnet/arcade-services/pull/1384

Reply:
Is that true?  That's not how I interpret https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#in

> Evaluates True if left parameter is equal to any right parameter

Reply:
It's not the in, it's the Build.SourceBranchName variable that strips the prefix 😢 

See https://dnceng.visualstudio.com/internal/_build/results?buildId=797060&view=results

Reply:
Ah, of course.  I read that section about sourceBranchName the other day, but I only read as much as I cared about.  Thanks, carry on.

Reply:
> I read that section about sourceBranchName the other day, but I only read as much as I cared about.

Exact same thing that happened to me when making the previous PR. Just outright missed the whole example:
>In refs/heads/feature/tools this value is tools.